ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-commands.sh

```diff
@@ -37,15 +37,15 @@ else
     echo "OO_INSTALL_NAMESPACE is '$OO_INSTALL_NAMESPACE'"
 fi
 
-if [[ -n $NS_NAMESTANZA ]]; then
+if [[ -n "$NS_NAMESTANZA" ]]; then
```

Comment:
@petr-muller Seems like it didn't take effect, still get the errors below. Do you know how to make this Shell file take effect after some updates? Thanks!
```console
OO_INSTALL_NAMESPACE is 'default'
/bin/bash: line 43: NS_NAMESTANZA: unbound variable
error: failed to execute wrapped command: exit status 1
```

Reply:
@benluddy I find you have made some contribution to this file. Do you happen to know that? Thanks very much!